src/shared/api/CivicAPI.ts

```diff
@@ -0,0 +1,265 @@
+import $ from 'jquery';
```

Comment:
There is no swagger definition for the Civic API. Is it that important to use `superagent` instead of `jQuery`?